packages/core/src/components/Sequence/SequenceFooter.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { FC, ReactNode } from 'react';
+import { Button, makeStyles } from '@material-ui/core';
+import { StepActions } from './Sequence';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'block',
+    marginTop: theme.spacing(3),
+    '& button': {
+      marginRight: theme.spacing(1),
+    },
+  },
+}));
+
+export const RestartBtn: FC<{
+  text?: string;
+  handleClick?: () => void;
+  stepIndex: number;
+}> = ({ text, handleClick }) => (
+  <Button onClick={handleClick}>{text || 'Reset'}</Button>
+);
+const NextBtn: FC<{
+  text?: string;
+  handleClick?: () => void;
+  disabled?: boolean;
+  last?: boolean;
+  stepIndex: number;
+}> = ({ text, handleClick, disabled, last, stepIndex }) => (
+  <Button
+    variant="contained"
+    color="primary"
+    disabled={disabled}
+    data-testid={`nextButton-${stepIndex}`}
+    onClick={handleClick}
+  >
+    {text || (last ? 'Finish' : 'Next')}
+  </Button>
+);
+const BackBtn: FC<{
+  text?: string;
+  handleClick?: () => void;
+  disabled?: boolean;
+  stepIndex: number;
+}> = ({ text, handleClick, disabled, stepIndex }) => (
+  <Button
+    onClick={handleClick}
+    data-testid={`backButton-${stepIndex}`}
+    disabled={disabled}
+  >
+    {text || 'Back'}
+  </Button>
+);
+
+export type SequenceFooterProps = {
```

Comment:
Since we're in `SequenceFooter.tsx`, I'd just call this `Props`

Reply:
it makes it easier to export and use in a test so i generally like to preface it with the filename